client_source/sqlplus/ut_run.sql

```diff
@@ -289,7 +355,25 @@ begin
     loop
       fetch :l_run_params_cur into l_reporter_id, l_reporter_name;
       exit when :l_run_params_cur%notfound;
-        p('  v_reporter := '||l_reporter_name||'();');
+        if lower(l_reporter_name) in ('ut_coveralls_reporter', 'ut_coverage_sonar_reporter', 'ut_coverage_html_reporter') then
+          p('  v_reporter := '||l_reporter_name||'( a_file_paths => ut_varchar2_list(');
+          loop
+            fetch :l_source_files into l_file_path;
+            exit when :l_source_files%notfound or l_file_path is null;
+            p('      '''||l_file_path||''',');
+          end loop;
+            p('      null));');
+        elsif lower(l_reporter_name) = ('ut_sonar_test_reporter') then
```

Comment:
It is not, as `ut_sonar_test_reporter` accepts list of file names that represent test code. The ones in `if` are passing file names for project sources (code under test).